learn.md

```diff
@@ -41,45 +41,45 @@ redirect_from:
 		<h3 id="generatinging-code-documentation"><a href="/learn/generating-code-documentation/">Generating Code Documentation
 </a></h3>
 		<p>The usage of the <code class="highlighter-rouge language-plaintext">bal doc</code> CLI command.</p>
 		<h3 id="making-ballerina-programs-configurable"><a href="/learn/making-ballerina-programs-configurable/defining-configurable-variables/">Making Ballerina Programs Configurable</a></h3>
 		<p>The language support for configurability.</p>
 		<h3 id="observing-ballerina-programs"><a href="/learn/observing-ballerina-programs/observing-your-application-with-prometheus-grafana-and-jaeger/">Observing Ballerina Programs
 </a></h3>
-		<p>The basics of the observing functionalities that are provided for Ballerina programs.</p>
+		<p>Basics of the observability functionalities that are provided for Ballerina programs.</p>
 		<h3 id="running-ballerina-programs-in-the-cloud"><a href="/learn/running-ballerina-programs-in-the-cloud/code-to-cloud/">Running Ballerina Programs in the Cloud
 </a></h3>
 		<p>The cloud offerings for running Ballerina programs.</p>
 		<h3 id="managing-dependencies"><a href="/learn/managing-dependencies/">Managing Dependencies </a></h3>
-		<p>Declare and manage dependencies and how to use the local repository.</p>
+		<p>Declaring and managing dependencies and using the local repository.</p>
 		<h3 id="publishing-packages-to-ballerina-central"><a href="/learn/publishing-packages-to-ballerina-central/">Publishing Packages to Ballerina Central</a></h3>
-		<p>Instructions on how to publish your package to Ballerina Central.</p>
+		<p>Details on publishing your library package to Ballerina Central.</p>
 		<h3 id="calling-java-code-from-ballerina-and-vice-versa"><a href="/learn/calling-java-code-from-ballerina-and-vice-versa/">Calling Java Code from Ballerina and Vice Versa</a></h3>
 		<p>Instructions on the supported interoperability features.</p>
 		</div>
 	</div>
 	<div class="column-gray-box-grid">
 		<div class="column-gray-box">  
 		<h2 id="references">References</h2>
-		<h3 id="language-introduction-slides"><a href="/learn/language-introduction/">Language Introduction</a></h3>
-		<p>Presentation slides on the Ballerina language that anyone can use to talk about the language.</p>
 		<!--<h3 id="language-guide"><a href="/learn/language-concepts/">Language Guide</a></h3>
 		<p>An elaborate textual guide to the Ballerina language.</p>-->
 		<h3 id="reference-guide-by-examples"><a href="/learn/by-example/">Reference by Examples</a></h3>
-		<p>A series of examples that serve as a reference guide for each language construct and concept.</p>
+		<p>A series of examples that serve as a reference guide for language constructs, concepts, and standard library modules.</p>
 		<h3 id="library-documentation"><a href="https://lib.ballerina.io/">Library Documentation</a></h3>
-		<p>All library API docs.</p>
+		<p>Standard library API documentation.</p>
```

Comment:
This is pointing to lib.ballerina.io where it contains all lib docs, not just standard library API docs. So this description is incorrect.

Reply:
Updated to: "API Documentation of all the Ballerina libraries."